app/assets/javascripts/pretty-text/addon/engines/discourse-markdown-it.js

```diff
@@ -344,9 +344,9 @@ function buildCustomMarkdownCookFunction(engineOpts, defaultEngineOpts) {
 function createMarkdownItEngineWithOpts(markdownitOpts, ruleOverrides) {
   if (ruleOverrides !== undefined) {
     // Preset for "zero", https://github.com/markdown-it/markdown-it/blob/master/lib/presets/zero.js
-    return window.markdownit("zero", markdownitOpts).enable(ruleOverrides);
+    return globalThis.markdownit("zero", markdownitOpts).enable(ruleOverrides);
```

Comment:
The `markdownit` function doesn't seem to be available on the `window` object anymore. Are we allowed to use `globalThis`? I think it's fine, because we check for it here: https://github.com/discourse/discourse/blob/275849771f32ede21072aea81cc2256247595e38/app/assets/javascripts/browser-detect.js#L1

But I had to make eslint allow its usage, so I'm not completely sure. :confused:

Reply:
If we already check for it then it's fine. I'll update the eslint config 🙂